src/internal/storage/fileset/unordered_writer.go

```diff
@@ -170,8 +170,7 @@ func newUnorderedWriter(ctx context.Context, storage *Storage, memThreshold int6
 	return uw, nil
 }
 
-// Append appends a file to the file set.
-func (uw *UnorderedWriter) Append(p string, overwrite bool, r io.Reader, customTag ...string) error {
+func (uw *UnorderedWriter) Put(p string, appendFile bool, r io.Reader, customTag ...string) error {
```

Comment:
Can we just separate `Put` and `Append`?

Reply:
Especially since this is an internal API lets just be explicit, there is no continuity we need to keep for users with regard to what Put does.

Reply:
The way I was thinking about this is that it is the entry point into the storage layer, so the interface lines up with what the client needs (PFS). I guess we could change the interface to Add / Delete like the fileset writer and PFS would have a helper function that converts a Put to the appropriate Add / Delete, but I kind of think what we have is fine. I think we should revisit this later because I don't think it is particularly important now.